### PR TITLE
Fix/shortcuts

### PR DESCRIPTION
**Pull request by jose1711**
### What does this PR do?

Update of in-game hnts related to latency tuning during playback to match the changes in code

### Closes Issue(s)

#454 

### Motivation

Hints should be kept up-to-date.

### CODE REVIEW

File: lang/performous.pot [hunk 1]
```diff
@@ -17,647 +17,361 @@ msgstr ""
...
-msgid ""
-"Override autodetection and force the fourth controller to the given type."
-msgstr ""
-
-#: ../data/config/schema.xml:146
-msgid "Joystick configuration"
-msgstr ""
-
-#: ../data/config/schema.xml:147
-msgid "Force joystick to some kind of instrument."
```
Comment (Baklap4) [inline lang/performous.pot]: Are these deletions valid?

**Comment by Baklap4** [comment]
also can you rebase this branch upon current master?
`git checkout master && git pull && git checkout fix/shortcuts && git rebase -i master && git push -f`

File: lang/performous.pot (hunk 1 above)
Comment (jose1711) [reply lang/performous.pot]: hmmm.. I used poedits feature to update pot from source files. But it seems xml is not recognized. How do you guys refresh `performous.pot` normally?
```
xgettext -o performous.pot -j ../game/*.cc ../data/config/schema.xml
xgettext: warning: file '../data/config/schema.xml' extension 'xml' is unknown; will try C
```

Comment (jose1711) [reply lang/performous.pot]: Found everything I needed in `README`, please see the updated commit

File: lang/performous.pot [hunk 2]
```diff
@@ -490,863 +575,3 @@ msgstr ""
...
-msgid "Continue"
-msgstr ""
-
-#: ../game/screen_playlist.cc:171
-msgid "Continue playing"
-msgstr ""
-
-#: ../game/screen_playlist.cc:179
-msgid "Add songs"
-msgstr ""
```
Comment (Baklap4) [inline lang/performous.pot]: There is still a lot of deletions.. did you compile with all options on?

Comment (jose1711) [reply lang/performous.pot]: poedit apparently ignores `*.cpp` files, maybe this update? (with help of `xgettext` inside console)

File: lang/performous.pot [hunk 3]
```diff
@@ -778,575 +871,434 @@ msgstr ""
...
-#: ../game/webserver.cc:313
-msgid "Successfully removed song from playlist"
-msgstr ""
-
-#: ../game/webserver.cc:314
-msgid "Failed removing song from playlist"
-msgstr ""
-
-#: ../game/webserver.cc:315
-msgid "Successfully changed position of song"
```
Comment (Baklap4) [inline lang/performous.pot]: Still some messages which are in current master..

Comment (jose1711) [reply lang/performous.pot]: Should be resolved now, thank you for your patience

**Comment by nieknooijens** [comment]
build failed on semaphore.

**Comment by Baklap4** [comment]
@nieknooijens that's due to ced not being properly checked out sometimes at semaphore :/

**Comment by Baklap4** [comment]
@jose1711 There are still a lot of messages removed within this revision, messages for which im sure they're in there

**Comment by jose1711** [comment]
> @jose1711 There are still a lot of messages removed within this revision, messages for which im sure they're in there

i think they're only renamed (webserver -> requesthandler)

```
➜  lang git:(fix/shortcuts) msgfmt -v --statistics performous.pot
performous.pot: 0 translated messages, 335 untranslated messages.
➜  lang git:(fix/shortcuts) git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
➜  lang git:(master) msgfmt -v --statistics performous.pot
performous.pot: 0 translated messages, 312 untranslated messages.
```

**Comment by Baklap4** [comment]
@jose1711 thanks for your contribution!
Can you update this pull request to latest master, we then can get this merged i think :)

**Comment by Baklap4** [comment]
Closing in favor of https://github.com/performous/performous/pull/511

**Comment by jose1711** [comment]
Sorry, couldn't get to it in a timely manner. And thanks for the fix!

**Pull request closed without merging**